branch-validation.yml

```diff
@@ -229,13 +227,13 @@
       when: master is not defined
 
     - name: floating ips are reachable
-      command: ping -c 5 {{ item.public_ip }}
+      command: ping -c 60 {{ item.public_ip }}
       with_items: fip_controllers.results + fip_compute.results
       changed_when: false
       delegate_to: localhost
       register: last_ping
       until: comp_ping|success
-      retries: 5
+      retries: 60
```

Comment:
Just to be sure, this is going to basically be a timeout of 60*10, or 600 seconds, which is 10 minutes. Is 10 minutes the time you're looking for here?